### PR TITLE
Feature: completed Wishlist page

### PR DESCRIPTION
**Pull request by koushikrp**
- Completed html and css code of wishlist page

### CODE REVIEW

**Review by shagunbagrecha — APPROVED** [review]
Looks Good To me!

**Review by aniketxparihar — APPROVED** [review]
Looks Good

File: src/pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+				</div>
+
+				<div class="card card-v floating-shadow">
+					<div class="card-body flex justify-center">
+						<div class="card-header flex justify-between">
+							<div class="card-badge">ON SALE</div>
+							<span><i class="fas fa-heart"></i></span>
+							<span class="hidden"><i class="fal fa-times-circle"></i></span>
+						</div>
+						<div class="card-overlay-text-div hidden">
```
Comment (shashidwivedi) [inline src/pages/wishlist.html:182]: Will suggest not to use the word 'div' in class names. Think of it as a reserved word (it's not actually)

**Review by shashidwivedi — APPROVED** [review]
The rest looks fine. Please see my other comment

File: src/pages/wishlist.html (hunk 1 above)
Comment (koushikrp) [reply src/pages/wishlist.html:182]: Sure Shashi, Will keep in mind. thanks.

**Pull request merged**